### PR TITLE
Added Homework exersizes

### PR DESCRIPTION
**Pull request by AnnieGrig**

### CODE REVIEW

File: src/lesson1/Lesson1Ex10.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package lesson1;
+
+import java.util.Scanner;
+
+public class Lesson1Ex10 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int w = 7;
```
Comment (Smbat888) [inline src/lesson1/Lesson1Ex10.java]: w-n u h-y mutq ara scanner-ov

File: src/lesson1/Lesson1Ex11.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package lesson1;
+
+public class Lesson1Ex11 {
+    public static void main(String[] args) {
+        int x = 7;
```
Comment (Smbat888) [inline src/lesson1/Lesson1Ex11.java]: x, y mutq ara scannerov, avelord commentneern el jnji

File: src/lesson1/Lesson1Ex12.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package lesson1;
+
+public class Lesson1Ex12 {
+    public static void main(String[] args) {
+        int x = 7;
```
Comment (Smbat888) [inline src/lesson1/Lesson1Ex12.java]: x, y, scanner-ov mutq ara

File: src/lesson1/Lesson1Ex2.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import java.util.Scanner;
+
+public class Lesson1Ex2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter first number: ");
+        int x = scanner.nextInt();
+        System.out.print("Enter second number: ");
+        int y = scanner.nextInt();
+        System.out.println(x + y);
```
Comment (Smbat888) [inline src/lesson1/Lesson1Ex2.java]: hnenc ara vor verjum tpi orinak:

5 + 7 = 12
kam
3 + 2 = 5
ev ayln

qo mot uxxaki arjeqna tpum

File: src/lesson1/Lesson1Ex3.java [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+import java.util.Scanner;
+
+public class Lesson1Ex3 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter first number: ");
+        double x = scanner.nextDouble();
+        System.out.print("Enter second number: ");
+        int y = scanner.nextInt();
+        System.out.println(x / y);
```
Comment (Smbat888) [inline src/lesson1/Lesson1Ex3.java]: nuynn el stex, menak arjeqna tpum, hnenc ara vor matematikakan artahaytutyunn el tpi:

5 / 1 = 5 
ev ayln..

File: src/lesson1/Lesson1Ex5.java [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package lesson1;
+
+public class Lesson1Ex5 {
+    public static void main(String[] args) {
+        int x = 10;
```
Comment (Smbat888) [inline src/lesson1/Lesson1Ex5.java]: x, y scannnerov mutq ara

File: src/lesson1/Lesson1Ex6.java [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+import java.util.Scanner;
+
+public class Lesson1Ex6 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int x = 8;
+        int y;
+        System.out.println("Enter the number 8");
+        x = sc.nextInt();
+        for(y = 1; y <= 10; y++){
```
Comment (Smbat888) [inline src/lesson1/Lesson1Ex6.java]: for (int y = 1; ....)

y-n imast chuni for-iic durs haytarares

u himnakanum for-i mej popoxakani anuny i, j, k,.. sranq en dnum, yndunvac praktika a, y anuny poxi i-ov

File: src/lesson1/Lesson1Ex8.java [hunk 8]
```diff
@@ -0,0 +1,14 @@
+package lesson1;
+
+public class Lesson1Ex8 {
+    public static void main(String[] args) {
+        float radius = 7.5f;
```
Comment (Smbat888) [inline src/lesson1/Lesson1Ex8.java]: radius-y mutq ara scannerov

File: src/lesson2/Lesson2Ex11.java [hunk 9]
```diff
@@ -0,0 +1,13 @@
+package lesson2;
+
+public class Lesson2Ex11 {
+    public static void main(String[] args) {
+        boolean doesSignificantWork = true;
```
Comment (Smbat888) [inline src/lesson2/Lesson2Ex11.java:5]: es popoxakannery scannerov mutq ara,
verjum el gone mi ban tpi imananq inch exav))

File: src/lesson2/Lesson2Ex12.java [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+        float x = sc.nextFloat();
+        System.out.println("Please enter second number");
+        float y = sc.nextFloat();
+        System.out.println("Please enter the operator");
+
+        Scanner symbolScaner = new Scanner(System.in);
+        String a = symbolScaner.nextLine();
+        char symbol = a.charAt(0);
+
+        if (symbol == '*') {
```
Comment (Smbat888) [inline src/lesson2/Lesson2Ex12.java]: if-else-elseeif-y poxi swith-case -ov

File: src/lesson2/Lesson2Ex3.java [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+import java.util.Scanner;
+
+public class Lesson2Ex3 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Please enter first number");
+        double a = sc.nextDouble();
+        System.out.println("Please enter second number");
+        double b = sc.nextDouble();
+                if (a >= 0 & a <= 1 && b >=0 & b <= 1) {
```
Comment (Smbat888) [inline src/lesson2/Lesson2Ex3.java]: & te && ?

File: src/lesson3/Lesson3Ex11.java [hunk 12]
```diff
@@ -0,0 +1,19 @@
...
+import java.util.Scanner;
+
+public class Lesson3Ex11 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the number: ");
+
+        int num = sc.nextInt();
+
+            if (num % 2 == 0 || num % 3 == 0)
```
Comment (Smbat888) [inline src/lesson3/Lesson3Ex11.java]: che ay, sennc chi eexni, ba vor 5-i bajanvec? kam 7-i, ev ayln..
sa petqa vor while-ov karenas anes

File: src/lesson3/Lesson3Ex2.java [hunk 13]
```diff
@@ -0,0 +1,17 @@
+package lesson3;
+
+public class Lesson3Ex2 {
+    public static void main(String[] args) {
+        int sum = 0;
+        System.out.println("For number 20 : ");
```
Comment (Smbat888) [inline src/lesson3/Lesson3Ex2.java]: 20-i hamar tiv mutq ara, tox usern input ani te minchev vor tivy fra gumari, tenc cragiry avelii funkcional klni

File: src/lesson3/Lesson3Ex5.java [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+        int y = sc.nextInt();
+        int sum = 0;
+        for (int i = x; i <= y; i++) {
+            if (i % 9 == 0) {
+                System.out.print( i + " ");
+                sum += i;
+            }
+        }
+
+        System.out.println("\nSum = " + sum);
```
Comment (Smbat888) [inline src/lesson3/Lesson3Ex5.java]: println-i depqum tanuma hajord tox, es depqum el imast chuni \n dnes stringit mej

File: src/lesson3/Lesson3Ex9.java [hunk 15]
```diff
@@ -0,0 +1,19 @@
...
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the number: ");
+
+        int num = sc.nextInt();
+        int sum = 1;
+
+        for (int i = num; i > 0; i--) {
+            sum *= i;
+        }
+            System.out.println(sum);
```
Comment (Smbat888) [inline src/lesson3/Lesson3Ex9.java]: tab-erin u coding style-in ushadir exi, save aneluc araj Alt+Ctr+L ara, to coding styled avtomat dzi inqy.

File: src/lesson2/Lesson2Ex3.java (hunk 11 above)
Comment (AnnieGrig) [reply src/lesson2/Lesson2Ex3.java]: [0,1] mijakayqi hamar erku taberakn el chisht chen ? kodi mej chisht a tpum ))

File: src/lesson2/Lesson2Ex11.java (hunk 9 above)
Comment (AnnieGrig) [reply src/lesson2/Lesson2Ex11.java:5]: chishtn asac es meki pahanjy yndhanrapes chem haskacel (((

Comment (AnnieGrig) [reply src/lesson2/Lesson2Ex11.java:5]: haskanum em inch a uzum, bayc chem haskanum inch anem

File: src/lesson3/Lesson3Ex11.java (hunk 12 above)
Comment (AnnieGrig) [reply src/lesson3/Lesson3Ex11.java]: 5-i u 7-i hamar chisht a ...bolor parzeri hamar chisht a bayc ste xndirn urish a 2-i u 3-i hamar a sxal .. menak et 2 tivn a vor parz a, bayc im gracov parzeri mej chi... mtacum em uxxaki et erku tvi hamar arandin payman grem

File: src/lesson2/Lesson2Ex11.java (hunk 9 above)
Comment (Smbat888) [reply src/lesson2/Lesson2Ex11.java:5]: mi hat uxarki xndiry estex:
copy ara pahanjy commentov gri asem

File: src/lesson2/Lesson2Ex3.java (hunk 11 above)
Comment (Smbat888) [reply src/lesson2/Lesson2Ex3.java]: es pahin chishta ashxatum, 
bayc &-y binar operacia a, isk &&-y conditional, qez &&-na petq, kap chuni vor hima senc ashxatuma.

File: src/lesson3/Lesson3Ex11.java (hunk 12 above)
Comment (Smbat888) [reply src/lesson3/Lesson3Ex11.java]: che che, sa while-ov ara, sra cchishty while-ov a!

File: src/lesson2/Lesson2Ex11.java (hunk 9 above)
Comment (AnnieGrig) [reply src/lesson2/Lesson2Ex11.java:5]: Consider the following if statement, where doesSignificantWork ,
makesBreakthrough , and nobelPrizeCandidate are all boolean variables:
if (doesSignificantWork) {
if (makesBreakthrough)
nobelPrizeCandidate = true;
else nobelPrizeCandidate = false;
} else if (!doesSignificantWork)
nobelPrizeCandidate = false;
First, write a simpler if statement that is equivalent to this one. Then write a single
assignment statement that does the same thing.

File: src/lesson2/Lesson2Ex3.java (hunk 11 above)
Comment (AnnieGrig) [reply src/lesson2/Lesson2Ex3.java]: ok haskaca ))

File: src/lesson3/Lesson3Ex11.java (hunk 12 above)
Comment (AnnieGrig) [reply src/lesson3/Lesson3Ex11.java]: lav hima kpordzem

File: src/lesson2/Lesson2Ex11.java (hunk 9 above)
Comment (Smbat888) [reply src/lesson2/Lesson2Ex11.java:5]: ha lov ok, esi esh bana mi xoraci :D

Comment (AnnieGrig) [reply src/lesson2/Lesson2Ex11.java:5]: ha indz el dur chekav :D

**Pull request merged**